test/components/attribute-picker/attribute-picker-item.vdiff.js

```diff
@@ -0,0 +1,61 @@
+import '../../../src/components/attribute-picker/attribute-picker-item.js';
+import { expect, fixture, hoverElem, html, waitUntil } from '@brightspace-ui/testing';
+
+describe('attribute-picker-item', () => {
+	it('default', async() => {
+		const component = await fixture(html`<d2l-labs-attribute-picker-item text="attribute"></d2l-labs-attribute-picker-item>`);
```

Comment:
Nit: many of these use the same fixture -- might want to pull its definition out into a `defaultFixture` variable at the top of the file or something.

Reply:
Don't think I've ever used or heard of `defaultFixture` before, so I'll have to read more about it

Reply:
Haha yeah exactly. Just like:

```javascript
const defaultFixture = html`<d2l-labs->...`;
```

Reply:
I created it as a function in the end, similar to what I'd done with the attribute picker test, so that it could also be used for the deletable state, while also resetting between tests 👍